pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "pm_icecon"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Until we release v1.0.0, minor version changes signal breaking changes. This is definitely a breaking-change PR!

Reply:
Agree!